ext_emconf.php

```diff
@@ -1,10 +1,12 @@
 <?php
 
+declare(strict_types=1);
+
 $EM_CONF[$_EXTKEY] = [
     'title' => 'News Page',
     'description' => 'Use pages as news records.',
     'category' => 'fe',
-    'version' => '2.0.2',
+    'version' => '3.0.0',
```

Comment:
I've prepared the version for the nextnv13 release already.

Reply:
i do this in general before tagging, to have an Release-Commit (just fyi)